client/src/GachaContainer.jsx

```diff
@@ -1,45 +1,40 @@
-import React, { Component } from 'react';
-import { Segment, Button } from 'semantic-ui-react';
-
-import Summon from './components/Summon';
-import Servant from './components/Servant';
+import React, { Component } from "react";
+import { Segment, CardGroup } from "semantic-ui-react";
+import Summon from "./components/Summon";
+import Servant from "./components/Servant";
 
 class GachaContainer extends Component {
-    state = {
-        summoned: false,
-        loading: false,
-        servant: {}
-    };
+  state = {
+    summoned: false,
+    loading: false,
+    servant: {},
+    number: 0
+  };
 
-    summonServant = () => {
-        const newSummon = !this.state.summoned;
-        this.setState({ summoned: newSummon, loading: true }, () => this.showServant());
-    }
+  summonServant = number => {
+    const newSummon = !this.state.summoned;
+    this.setState({
+      summoned: newSummon,
+      loading: false,
```

Comment:
You think we can use Semantic UI's Placeholder component here?

Reply:
Will look into it.